src/components/SplitView/ToggleButton.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import { IconSVG } from '../Icon';
+
+export const TOGGLE_BUTTON_WIDTH = '0.75rem';
+
+export const ToggleButton = ({
+  ariaControls,
+  assistiveTextOpen,
+  assistiveTextClose,
+  isOpen,
+  onClick,
+}) => {
+  const assistiveText = isOpen
+    ? assistiveTextOpen
+    : assistiveTextClose;
+  const buttonClassName = cx(
+    'slds-button slds-button_icon slds-split-view__toggle-button',
+    { 'slds-is-open': isOpen },
+  );
+
+  return (
+    <button
```

Comment:
is it not possible to use an `IconButton` here? This seems to just reimplement a good portion of that component.

Reply:
Icon button brings slds styles, that break visuals.